packages/byods/src/Errors/catalog/BYODSDeviceError.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable valid-jsdoc */
+//import {RegistrationStatus} from '../../common/types';
+
+import {ErrorContext, ErrorMessage, ErrorObject, ERROR_TYPE} from '../types';
+import ExtendedError from './ExtendedError';
+
+/**
+ * Any error reported from Calling client should be stored here.
+ */
+export class BYODSError extends ExtendedError {
+  // public status: RegistrationStatus = RegistrationStatus.INACTIVE;
```

Comment:
Do we need this comment? I notice a lot of comments in the file. Can we remove the ones which are not required?

Reply:
@adhmenon Updated...!